platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/reconnect/StateBuilder.java

```diff
@@ -67,8 +71,13 @@ public void buildState(
             final double teacherRemoveProbability,
             final double teacherModifyProbability,
             final BiConsumer<K, V> teacherPopulator,
-            final BiConsumer<K, V> learnerPopulator) {
+            final BiConsumer<K, V> learnerPopulator,
+            final Consumer<Long> storageOptimizer) {
+        System.err.printf("Building a state of size %,d\n", size);
```

Comment:
Debug output?

Reply:
Same as above. This is left here on purpose.